src/sentry/static/sentry/app/components/assistant/supportDrawer.jsx

```diff
@@ -0,0 +1,133 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import createReactClass from 'create-react-class';
+import $ from 'jquery';
+import HookStore from '../../stores/hookStore';
+
+// SupportDrawer slides up when the user clicks on a "Need Help?" cue.
+const SupportDrawer = createReactClass({
+  displayName: 'SupportDrawer',
+
+  propTypes: {
+    closeHandler: PropTypes.func.isRequired,
+  },
+
+  getInitialState() {
+    return {
+      inputVal: '',
+      docResults: [],
+      helpcenterResults: [],
+    };
+  },
+
+  componentWillReceiveProps(props) {
+    this.setState({inputVal: ''});
+  },
+
+  handleSubmit(evt) {
+    evt.preventDefault();
+  },
+
+  handleChange(evt) {
+    evt.preventDefault();
+    let term = encodeURIComponent(evt.currentTarget.value);
+
+    this.setState({
+      inputVal: evt.currentTarget.value,
+    });
+
+    if (term == '') {
+      return;
+    }
+    $.get(
+      `https://rigidsearch.getsentry.net/api/search?q=${term}&page=1&section=hosted`,
+      data => {
+        this.setState({docResults: data.items});
+      }
+    );
+    $.get(
+      `https://sentry.zendesk.com/api/v2/help_center/articles/search.json?query=${term}`,
+      data => {
+        this.setState({helpcenterResults: data.results});
+      }
+    );
+  },
+
+  renderDocsResults() {
+    return this.state.docResults.map((result, i) => {
+      let {title} = result;
+      let link = `https://docs.sentry.io/${result.path}/`;
+
+      return (
+        <a href={link} target="_blank" key={i + 'doc'}>
+          <li className="search-tag search-tag-docs search-autocomplete-item">
+            <span className="title">{title}</span>
+          </li>
+        </a>
+      );
+    });
+  },
+
+  renderHelpCenterResults() {
+    return this.state.helpcenterResults.map((result, i) => {
+      return (
+        <a href={result.html_url} target="_blank" key={i}>
```

Comment:
We have an `ExternalLink` component for this. Also shouldn't the <a> tag be inside the <li>?